testing/simapp/app.go

```diff
@@ -17,15 +17,6 @@ import (
 	"cosmossdk.io/core/appmodule"
```

Comment:
can we delete this app now?

Reply:
We still use it for the integration tests, I guess?

Reply:
ah, I guess this means we will need to keep it around so we don't have ibc-go importing from the simapp :thinking: ?

Reply:
Yeah, I think so. It has been trimmed down, but need this still.